godot-macros/src/docs.rs

```diff
@@ -122,6 +123,27 @@ fn siphon_docs_from_attributes(doc: &[Attribute]) -> impl Iterator<Item = String
         })
 }
 
+fn xml_escape(value: String) -> String {
+    if !value.contains(&['&', '<', '>', '"', '\'']) {
+        return value;
+    }
```

Comment:
To make sure this isn't refactored away, could you maybe add a short comment why we have this check first?

Reply:
Not sure about wording, but tried to add something